.gitignore

```diff
@@ -112,3 +112,4 @@ docs/index.rst
 
 # exclude project specific files
 pyuvsim/GIT_INFO
+pyuvsim/data/temporary_test_data/*
```

Comment:
This is fine, but I should note that `pytest` has a fixture for creating proper `tmp` directories, which we should probably migrate to.

Reply:
yeah, I need to read more about that. This is the existing structure, but the folder wasn't excluded via the gitignore (it was deleted at the end of the test session, but I noticed the files in my git status while tests were running so added it to the ignore for good measure).

Reply:
I'll create an issue for it.